src/Services/MeilisearchService.php

```diff
@@ -346,4 +383,20 @@ private function assertIsSearchable(string $className): void
             throw new Exception('Class '.$className.' is not searchable.');
         }
     }
+
+    /**
+     * @return class-string
+     */
+    private function getClassNameFromIndex(string $index): string
```

Comment:
This seems to be a common thing in the bundle, no? I'm wondering if this could be reused somewhere else.

Reply:
Yes, but I'd leave that for later :)